data/os/Debian/11.yaml

```diff
@@ -0,0 +1,4 @@
+---
+snmp::snmpd_options: "-Lsd -Lf /dev/null -u Debian-snmp -g Debian-snmp -I -smux,mteTrigger,mteTriggerConf -f -p /run/snmpd.pid"
+snmp::varnetsnmp_group: 'Debian-snmp'
+snmp::varnetsnmp_owner: 'Debian-snmp'
```

Comment:
These settings should be moved to the toplevel Debian hiera data. Debian-snmp has been the default user and group for snmp for at least the last 3 releases (probably longer than that). The current user/group settings are actually broken.